catroid/src/test/java/org/catrobat/catroid/test/content/bricks/CloneBrickUpdateVariableTest.java

```diff
@@ -69,8 +69,8 @@
 				{"HideTextBrick", new HideTextBrick()},
 				{"ShowTextBrick", new ShowTextBrick()},
 				{"ShowTextColorSizeAlignmentBrick", new ShowTextColorSizeAlignmentBrick()},
-				{"WebRequestBrick", new WebRequestBrick()},
 				{"ReadVariableFromDeviceBrick", new ReadVariableFromDeviceBrick()},
+				{"WebRequestBrick", new WebRequestBrick()},
```

Comment:
This change is not necessary but is also not faulty. 
Altogether the code looks nice! 
Eventually, you have to squash the commits, so please undo this change.